package.json

```diff
@@ -83,24 +82,22 @@
     "ui:lint:fix": "stylelint --config ./src/clr-core/.stylelintrc --fix \"src/clr-ui/**/*.scss\"",
     "core:start": "npm run core:sass && npm run core:api && npm-run-all --parallel core:sass:components:watch core:dev:watch",
     "core:dev:watch": "npm run core:sass && npm run core:api:json && ng serve dev-core",
-    "core:dev:watch:prod": "npm run core:sass && ng serve dev-core --prod",
+    "core:dev:watch:prod": "ng serve dev-core --prod",
     "core:dev:build": "ng build dev-core --prod",
-    "core:test": "npm run core:sass && ng test clr-core --watch=false && npm run core:golden:test",
+    "core:test": "npm run core:sass && ng test clr-core --watch=false",
     "core:test:watch": "ng test clr-core --watch --code-coverage",
-    "core:build": "npm-run-all core:lint core:sass:components core:build:ngpackagr core:build:package core:sass core:api:editor core:api core:golden core:build:package:format",
+    "core:test:bundle": "webpack --config src/clr-core/test-bundles/webpack.config.js && bundlesize --config src/clr-core/test-bundles/bundlesize.json",
+    "core:build": "npm-run-all core:lint core:sass:components core:build:ngpackagr core:build:package core:sass core:api:editor core:api core:build:package:format",
+    "core:build:watch": "npm run core:sass && ng build clr-core --watch",
     "core:build:package": "cpy ./src/clr-core/README.md ./dist/clr-core/; replace '@VERSION' $npm_package_version ./dist/clr-core/package.json",
     "core:build:package:format": "node ./src/clr-core/package.js",
     "core:build:ngpackagr": "ng build clr-core",
-    "core:golden": "ts-api-guardian --out dist/golden/clr-core.d.ts dist/clr-core/index.d.ts",
-    "core:golden:fix": "npm run core:sass && npm run core:build:ngpackagr && ts-api-guardian --out golden/clr-core.d.ts dist/clr-core/index.d.ts",
-    "core:golden:test": "diff -u golden/clr-core.d.ts dist/golden/clr-core.d.ts",
```

Comment:
We're getting rid of the golden file for core?

Reply:
It wasnt checking the entire API properly so we will have to come up with a new strategy for Core. I have some ideas but will attempt it in a different PR.